src/gui/mainwindow.cpp

```diff
@@ -251,6 +261,18 @@ MainWindow::MainWindow(QWidget *parent)
 
     m_prioSeparator = m_ui->toolBar->insertSeparator(m_ui->actionTopPriority);
     m_prioSeparatorMenu = m_ui->menuEdit->insertSeparator(m_ui->actionTopPriority);
+    
+#ifdef Q_OS_MAC
+    foreach (QAction *action, m_ui->toolBar->actions()) {
+        if (action->isSeparator()) {
```

Comment:
This removes vertical separators from the toolbar menu, they are invalid in macOS design. As I said above, there is no such concept. One could have space separation or no separation.

I replaced the separators with some extra space, but I think they are fine even if just removed.